packages/apps/public/locales/en/apps-config.json

```diff
@@ -13,5 +13,246 @@
   "rpc.header.test": "Test networks",
   "rpc.header.westend.relay": "Test Westend & parachains",
   "rpc.hosted.via": "via {{host}}",
-  "ss58.default": "Default for the connected node"
+  "rpc.kusama.altair": "Altair",
```

Comment:
I'm really not sure where these come from - the text for the chain name are not translated (anymore), so it should not build this at all.

Reply:
I would prefer not having these changes in - the reason being that record removal for these is (still) a manual process, so adding it means cleanup for me :)

Reply:
Oh perhaps I misunderstood. Yes of course will remove. Though I’m not quite sure what I need to remove because the tests passed.

Reply:
have removed the individual name chains that were added in replace for the `"ss58.default": "Default for the connected node"`